robot/EDA/resources/EDA.py

```diff
@@ -559,3 +559,32 @@ def verify_app_exists(self,app):
         """Verifies that the given app is present in the app launcher"""
         locator=eda_lex_locators["eda_settings"]["app_tile"].format(app)
         self.selenium.wait_until_page_contains_element(locator,timeout=60,error=f'{app} did not open in 1 min')
+
+    @capture_screenshot_on_error 
+    def checkbox_status(self,cbx_name,status):
+        """verifies if the specified checkbox is with expected status in readonly mode"""
+        locator=locator(cbx_name,status)
```

Comment:
This doesn't look right. I don't see a function named `locator` anywhere in the code. Was this supposed to be `locator=self.get_eda_locator(cbx_name, status)` or something like that?

Reply:
This method is not being used anywhere. Also, not sure what it was intended to do since it is just waiting on a locator. Hence, deleted this.